workflow-templates/im-test-k6-ci.properties.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Test - K6 Workflow for daily CI",
+    "description": "Workflow template for running K6 performance tests with every merge to master",
```

Comment:
Sorry I missed this one before.
```suggestion
    "description": "Workflow template for running K6 performance tests with every merge to main",
```

Reply:
I think this one does need to change, since we're running it for the main branch.